centreon/www/front_src/src/Main/index.tsx

```diff
@@ -55,8 +55,11 @@ const Main = (): JSX.Element => {
   const navigateTo = (path: string): void => {
     navigate(path);
     window.location.reload();
+    return 'string';
   };
 
+  console.log('hello darkness my old friend');
```

Comment:
🚫 **[Biome]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Don't use console
```
  59 │   };
  60 │
> 61 │   console.log('hello darkness my old friend');
     │   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  62 │
  63 │   startTransition(() => {
```
i Usage of console is disallowed.
i Unsafe fix: Remove console
```
   58  58 │       return 'string';
   59  59 │     };
   60     │ -
   61     │ - ··console.log('hello·darkness·my·old·friend');
   62  60 │
   63  61 │     startTransition(() => {
```
<!-- __reviewdog__:ChA3ZWUwM2MwNmQwMDFiMmNmEgVCaW9tZQ== -->